skymp5-front/src/components/SkyrimInput/styles.scss

```diff
@@ -1,8 +1,10 @@
-.skymp-input1 {
+.skymp-input {
   border: 3px solid rgba(157, 158, 158, 0.75);
```

Comment:
You should update SkyrimButton className in `index.js` and `styles.scss` as well.

Reply:
Right now SkyrimInput and SkyrimButton have similar className - `skymp-input`